registrations/migrations/0033_registration_price_group_vat.py

```diff
@@ -0,0 +1,85 @@
+# Generated by Django 3.2.23 on 2024-01-02 11:31
+
+from decimal import Decimal
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("registrations", "0032_alter_signuppricegroup_registration_price_group"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="registrationpricegroup",
+            name="price_without_vat",
+            field=models.DecimalField(
+                decimal_places=4, default=Decimal("0"), max_digits=19
+            ),
+        ),
+        migrations.AddField(
+            model_name="registrationpricegroup",
+            name="vat",
+            field=models.DecimalField(
+                decimal_places=4, default=Decimal("0"), max_digits=19
+            ),
+        ),
```

Comment:
This is not super critical since price groups are not in the production, but in theory these values would be nice to migrate for the existing price groups. At the moment the default value is used for them

Reply:
Added a data migration for VAT prices to this migration file